### PR TITLE
RLI - 52835

### PR DESCRIPTION
**Pull request by sandeephram**

### CODE REVIEW

**Comment by sharanyab16** [comment]
Jenkins please retry a build

**Comment by sharanyab16** [comment]
Jenkins please retry a build

File: community_supplied/Subscriber-Service/service.aaa-monitor-radius-server-delay.rule [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        rule monitor-radius-server-delay {
+            keys server-address;
+            synopsis "Monitors RADIUS server delay";
+            description "Monitors RADIUS server delay from the BNG's perspective against a provided baseline";
+            sensor radius-server-information {
+                synopsis "Telemetry for RADIUS server response-time measurement";
+                description "Telemetry for RADIUS server response-time measurement";
+                open-config {
+                    sensor-name /junos/system/subscriber-management/aaa/radius-servers/radius-server;
+                    frequency 30s;
```
Comment (gvarao) [inline community_supplied/Subscriber-Service/service.aaa-monitor-radius-server-delay.rule:12]: See if frequency can be changed to 60s in all rules. It is just recommendation by keeping performance in mind

File: community_supplied/Subscriber-Service/service.dhcp-relay-monitor-dhcpv4-server-delay.rule [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        rule monitor-dhcpv4-server-delay {
+            keys [ ri-name server-ip ];
+            synopsis "Monitor delay between a JUNOS BNG DHCPv4 relay and its DHCPv4 servers";
+            description "Monitors response time delay between the BNG acting as a DHCPv4 Relay and its DHCPv4 servers.";
+            sensor dhcpv4-data {
+                synopsis "DHCPv4 telemetry data";
+                description "Telemetry data for DHCPv4 component on the BNG";
+                open-config {
+                    sensor-name /junos/system/subscriber-management/client-protocols/dhcp/v4/;
+                    frequency 30s;
```
Comment (gvarao) [inline community_supplied/Subscriber-Service/service.dhcp-relay-monitor-dhcpv4-server-delay.rule:12]: Same comment as previous rule

**Review by gvarao — CHANGES_REQUESTED** [review]

**Review by gvarao — APPROVED** [review]

**Pull request merged**